ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -485,6 +485,82 @@ func (e *ExecutableHandlerImpl) validate() bool {
 	return true
 }
 
+func executeScript(script string) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	// Execute the shell script
+
+	cmd := exec.Command("bash", "-c", script)
```

Comment:
Will this work without bash? when we run this from container bash will not be available by default.

Reply:
Changed this to sh